### ISSUE TITLE
[terra-clinical-label-value-view] Insufficient color contrast

### ISSUE THREAD

**Issue by rakshith91** [cerner/terra-clinical#306]
# Issue Description
Ensure the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

Element has insufficient color contrast of 3.06 (foreground color: #909496, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1

Impact: serious

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Current Behavior
The current color of the label does not pass the WCAG AA contrast ratio.

**Comment by ryanthemanuel** [comment]
Once this is addressed we will want to update the TODO's introduced with https://github.com/cerner/terra-clinical/pull/307 in the wdio tests.

**Comment by neilpfeiffer** [comment]
This will be resolved with Issue #258 and PR #302, changing the font color for the label from `#909496` to `#75767d`
